x-pack/plugins/security_solution/public/common/components/event_details/alert_summary_view.test.tsx

```diff
@@ -116,6 +116,40 @@ describe('AlertSummaryView', () => {
       expect(getByText(fieldId));
     });
   });
+
+  test('DNS event renders the correct summary rows', () => {
+    const renderProps = {
+      ...props,
+      data: [
+        ...(mockAlertDetailsData.map((item) => {
+          if (item.category === 'event' && item.field === 'event.category') {
+            return {
+              ...item,
+              values: ['dns'],
+              originalValue: ['dns'],
+            };
+          }
+          return item;
+        }) as TimelineEventsDetailsItem[]),
+        {
+          category: 'dns',
+          field: 'dns.question.name',
```

Comment:
Does it make sense to also display `dns.question.type` in the flyout summary for parity with the DNS row renderer, show in the screenshot below?

![dns-row-renderer](https://user-images.githubusercontent.com/4459398/153253545-21d4d72a-7cc0-4ca5-84aa-bbd8db6ce4a5.png)

CC @paulewing